src/java.desktop/share/classes/sun/awt/image/ImageRepresentation.java

```diff
@@ -109,7 +109,8 @@ public synchronized void reconstruct(int flags) {
             src.checkSecurity(null, false);
         }
         int missinginfo = flags & ~availinfo;
-        if ((availinfo & ImageObserver.ERROR) == 0 && missinginfo != 0) {
+        if ((availinfo & (ImageObserver.ERROR | ImageObserver.FRAMEBITS)) == 0
+             && missinginfo != 0) {
```

Comment:
```suggestion
        if ((availinfo & (ImageObserver.ERROR | ImageObserver.FRAMEBITS)) == 0
            && missinginfo != 0) {
```
Nit: the wrapped line should be aligned to the opening parenthesis of the `if`, otherwise it looks as if the wrapped condition is part of the condition in the parentheses but it is not.